### PR TITLE
ft(S3C-4100): Add task for manual metric adjustment

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: libV2/tasks/ManualAdjust.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    async _execute() {
+        const timestamp = Date.now();
+
+        const objectDelta = this._program.objects;
+        const sizeDelta = this._program.storage;
+
+        if (!objectDelta && !sizeDelta) {
+            throw Error('You must provided at least one of --objects or --storage');
+        }
```
Comment (alexanderchan-scality) [inline libV2/tasks/ManualAdjust.js]: correct me if I am wrong, `--storage` and `--objects` are required options, so this check may be redundant.

Comment (tmacro) [reply libV2/tasks/ManualAdjust.js]: I must have discovered `requiredOption` after this

**Review by alexanderchan-scality — APPROVED** [review]

**Review by ilkescality — APPROVED** [review]

**Comment by rahulreddy** [comment]
Bypassing build status as there is an issue with CDN for docker image

**Pull request merged**